Autoload.php

```diff
@@ -152,8 +178,9 @@ private function prepareIncludePath()
             }
         }
         
-        set_include_path(implode(PATH_SEPARATOR, $mageAppIncludeDirs));
-        $this->manualIncludeDirs = array_merge($vendorIncludeDirs, $mageLibIncludeDirs, $globalIncludeDirs);
+        $autoIncludeDirs = array_merge($mageAppIncludeDirs, $vendorIncludeDirs, $mageLibIncludeDirs);
+        set_include_path(implode(PATH_SEPARATOR, $autoIncludeDirs));
+        $this->manualIncludeDirs = $globalIncludeDirs;
         
```

Comment:
There is no change to that method it's just git being weird. Don't know why it showed a diff :(